### ISSUE TITLE
The following error has occurred

### ISSUE THREAD

**Issue by VF128S** [careteditor/caret#153]
The following error has occurred:
```
{
    "message": "Attempting to call a function in a renderer window that has been closed or released. Function provided here: Updates.js:5:3414.",
    "version": "1.7.2",
    "platform": "darwin",
    "os": "15.6.0",
    "stack": [
        "Error: Attempting to call a function in a renderer window that has been closed or released. Function provided here: Updates.js:5:3414.",
        "    at AutoUpdater.callIntoRenderer (/Applications/Caret.app/Contents/Resources/electron.asar/browser/rpc-server.js:190:19)",
        "    at emitOne (events.js:101:20)",
        "    at AutoUpdater.emit (events.js:188:7)"
    ]
}
```

**Comment by astoilkov** [comment]
Hi,

Can you describe what were you doing when this error occurred?

Thanks in advance.

**Comment by VF128S** [comment]
ok,I open a md file that only have 1 url inside,copy it,open the chrome 
paste and i find the copy fail,so i return to caret,select the url again
then caret crash

i'm not sure if it conflicts with popclip

**Comment by astoilkov** [comment]
Does this happen persistently, from time to time or just once? Additionally, have you updated the app to version 1.8.0 and does the issue still replicate there?

**Comment by VF128S** [comment]
no just once

**Comment by astoilkov** [comment]
Ok. We found the cause of this issue and fixed it. The fix is now available in the latest 1.8.1 Caret release.

Closing this and thanks for reporting.
